docs/references/protocol/ledger-data/ledger-entry-types/oracle.md

```diff
@@ -0,0 +1,90 @@
+  "PriceDataSeries": [
+    {
+      "PriceData": {
+        "BaseAsset": "XRP",
+        "QuoteAsset": "USD",
+        "AssetPrice": 74,
+        "Scale": 2,
+      }
+    },
+  ],
+  "LastUpdateTime": 743609414,
+  "PreviousTxnID": "C53ECF838647FA5A4C780377025FEC7999AB4182590510CA461444B207AB74A9",
+  "PreviousTxnLgrSeq": 56865244
+}
+```
+
+
+## Oracle Fields
+
+| Field               | JSON Type | Internal Type | Required? | Description |
+|---------------------|-----------|---------------|-----------|-------------|
+| `Owner`             | String    | AccountID     | Yes       | The XRPL account with update and delete privileges for the oracle. It's recommended to set up [multi-signing](https://xrpl.org/set-up-multi-signing.html) on this account. |
+| `Provider`          | String    | Blob          | Yes       | An arbitrary value that identifies an oracle provider, such as Chainlink, Band, or DIA. This field is a string, up to 256 ASCII hex encoded characters (0x20-0x7E). |
+| `PriceDataSeries`   | Array     | Array         | Yes       | An array of up to 10 `PriceData` objects, each representing the price information for a token pair. More than five `PriceData` objects require two owner reserves. |
+| `LastUpdateTime`    | Number    | UInt32        | Yes       | The time the data was last updated, represented in Unix time. |
+| `URI`               | String    | Blob          | No        | An optional Universal Resource Identifier to reference price data off-chain. This field is limited to 256 bytes. |
+| `AssetClass`        | String    | Blob          | Yes       | Describes the type of asset, such as "currency", "commodity", or "index". This field is a string, up to 16 ASCII hex encoded characters (0x20-0x7E). |
+| `OwnerNode`         | String    | UInt64        | Yes       | A hint indicating which page of the oracle owner's owner directory links to this entry, in case the directory consists of multiple pages. |
+| `PreviousTxnID`     | String    | UInt256       | Yes       | The hash of the previous transaction that modified this entry. |
+| `PreviousTxnLgrSeq` | String    | UInt32        | Yes       | The ledger index that this object was most recently modified or created in. |
+
+
+### PriceData Fields
+
+| Field               | JSON Type | Internal Type | Required? | Description |
+|---------------------|-----------|---------------|-----------|-------------|
+| `BaseAsset`         | String    | Currency      | Yes       | The primary asset in a trading pair. Any valid identifier, such as a stock symbol, bond CUSIP, or currency code is allowed. For example, in the BTC/USD pair, BTC is the base asset; in 912810RR9/BTC, 912810RR9 is the base asset. |
+| `QuoteAsset`        | String    | Currency      | Yes       | The quote asset in a trading pair. The quote asset denotes the price of one unit of the base asset. For example, in the BTC/USD pair, USD is the quote asset; in 912810RR9/BTC, BTC is the quote asset. |
```

Comment:
```suggestion
| `BaseAsset`         | String    | Currency      | Yes       | The primary asset in a trading pair. For example, in the BTC/USD pair, BTC is the base asset; in 912810RR9/BTC, 912810RR9 is the base asset. See below for details on the currency code format. |
| `QuoteAsset`        | String    | Currency      | Yes       | The quote asset in a trading pair. The quote asset denotes the price of one unit of the base asset. For example, in the BTC/USD pair, USD is the quote asset; in 912810RR9/BTC, BTC is the quote asset. See below for details on the currency code format. |
```

I think it would be helpful to point people at the section below explaining the currency code format with an example below, and then move the details there rather than trying to fit it all into the table.